css/components/card_badge.css

```diff
@@ -0,0 +1,48 @@
+.card_with_badges {
+  height: 36rem;
+  width: 28rem;
+  position: relative;
+  background-image: linear-gradient(to bottom, transparent, var(--color-black)),
+    url("https://source.unsplash.com/random/500x500");
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
Can use shorthand here

Reply:
Yes may be but it would get more confusing so just to keep it simple I did it in this way 😊